### PR TITLE
Add create-repo.sh script to create in-cluster Helm repo

### PR DESCRIPTION
**Pull request by adriansuarez**
This change adds a script that creates a Helm repo that is usable within the local Kubernetes cluster that contains all published versions along with the development version packaged from the current commit.

### CODE REVIEW

**Comment by adriansuarez** [comment]
@sivanov-nuodb This is just copying what is in `operator/config/repo` in the nuodb-control-plane repo and running it as a script instead of a make target.

File: scripts/repo/charts/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (kontaras) [inline scripts/repo/charts/index.html:1]: Isn't this file getting overriden in `create-repo.sh`?
`curl -s https://nuodb.github.io/nuodb-helm-charts/index.yaml > repo/charts/index.yaml`

So, it should be in `.gitignore`.

**Review by kontaras — APPROVED** [review]

File: scripts/repo/charts/index.html (hunk 1 above)
Comment (adriansuarez) [reply scripts/repo/charts/index.html:1]: I'm going to adjust the readiness probe so that this file is not required.

**Review by sivanov-nuodb — APPROVED** [review]
Thanks for adding this into NuoDB Helm charts repo!

**Pull request merged**